arangodb-net-standard.Test/UserApi/UserApiClientTest.cs

```diff
@@ -198,6 +198,22 @@ public async Task GetUserAsync_ShouldThrow_WhenUserDoesNotExist()
             Assert.Equal(1703, ex.ApiError.ErrorNum); // ERROR_USER_NOT_FOUND
         }
 
+        [Fact]
+        public async Task GetUsersAsync_ShouldSucceed()
+        {
+            GetUsersResponse response = await _userClient.GetUsersAsync();
```

Comment:
Regarding testing the admin case, the documentation says the following:

> Fetches data about all users. You need the Administrate server access level in order to execute this REST call. Otherwise, you will only get information about yourself.

https://www.arangodb.com/docs/stable/http/user-management.html#list-available-users

In any cases, we get information back in an identical format. Here we just want to check that the endpoint and response are working properly rather than testing all cases, so I would say this test case is enough.

For the number of users, we don't know exactly what number of users to expect (e.g. tests are creating/deleting users), hence the check for `Count() > 0` which I think is enough.

> we could delete the users and catch the exception

Not sure what you mean. We already have tests to cover the delete scenario.